### ISSUE TITLE
Rev | 2 ssp: Biome 146

### ISSUE THREAD

**Issue by Krysommor** [Haggle1996/RevolutionPack#212]
[crash-2016-02-05_22.50.52-client.txt](https://github.com/Haggle1996/RevolutionPack/files/119976/crash-2016-02-05_22.50.52-client.txt)
Seed : 1895761144564026249
I was just looking around the spawn area (flying) making sure that there is nothing that could cause a problem then i came across this biome (north-west of spawn), it would have made a good addition to the other but this happened.

**Comment by devanchya** [comment]
Run into this issue on our new map as well.  Trying to figure out a temp fix.

**Comment by Aivech** [comment]
> No realistic version of biome X - A lot of people have been experiencing crashes that contain this
> message: "java.lang.RuntimeException: No realistic version of biome X", where X is a biome ID.
> This crash is usually caused by either a biome ID conflict of some sort, or by a mod that adds a
> biome which RTG doesn't support. Here are some things you can do to help prevent this crash
> from happening:
> 
> > - When configuring biome IDs, try to avoid using IDs 128 through 167, as these are reserved for
> >   vanilla's 'mutated' biomes and can cause strange behaviour even when mutated biomes are disabled.
> >   (You should also avoid ID 255 if possible.)

From https://github.com/Team-RTG/Realistic-Terrain-Generation/issues/466 last post.

Change the ID of whatever biome is 146 so that it does not fall inside that range.

**Comment by Haggle1996** [comment]
I'm certain it's not a biome conflict issue. BTW, 146 is the default biome AE2 specifies. If it were an issue, I'm sure it would have been reported to AE decades ago.

**Comment by Krysommor** [comment]
i will see if a dev is on a Ts i go on during the week to talk to him about the issue

**Comment by Aivech** [comment]
I am also sure it isn't a biome conflict.

**Comment by oxinabox** [comment]
@devanchya WRT temp fix:
what I posted in https://github.com/Team-RTG/Realistic-Terrain-Generation/issues/466
works:

Open: `config/AppliedEnergistics2.cfg` and disable Spatial IO.
I think it is the under networkfeatures
        `B:SpatialIO=false`

You won't have SpatialIO but you won't have crash,
and don't have to regen world.

---

@Haggle1996 sorry for not double posting this here, when I first found it.
I was really busy (and wanted to get back to spending my rare downtime actually playing the pack)
I figured you would find it when googling 146 missing anyway.
